federation-2/harmonizer/RELEASE_CHECKLIST.md

```diff
@@ -23,8 +23,13 @@ None of the `federation-rs` packages currently maintain changelogs as they are l
 
 1. Once merged, run `git checkout main && git pull`
 1. Run `cargo xtask tag --package $PUBSLUG --real-publish`
+   - **NOTE** If you get an error, you might need to run `git fetch --tags -f` to ensure the remote tags match the local tags
 1. Wait for CI to build and publish `harmonizer` to crates.io and `supergraph` to `federation-rs` GitHub releases.
 
+### Tag the `latest-2` release so rover automatically downloads the new version
+
+1. Run `git tag -d composition-latest-2 && git tag -a composition-latest-2 -m v{version} && git push --tags -f` replacing `{version}` with the version of composition.
```

Comment:
this is great to document - I think it only existed in slack DMs before. thank you for doing so. if you wanna take a crack at #80 we could make this a part of `xtask tag`? Can do in a follow-up PR for sure, I'd be happy to help out w/reviews and/or pairing if you'd like!

Reply:
i left [a comment](https://github.com/apollographql/federation-rs/issues/80#issuecomment-1144225559) on the approach i think we would want

Reply:
Sweet! I'd be down to take a stab at that, though in a different PR.